<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QWidget" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>268</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>441</width>
    <height>268</height>
   </size>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Filter</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="ui_treeView">
            <property name="text">
             <string>Tree View</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">ui_buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ui_CurrentDiagram">
            <property name="text">
             <string>&amp;Current diagram</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">ui_buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ui_Diagrams">
            <property name="text">
             <string>&amp;All diagrams</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">ui_buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder>ui_Diagrams</zorder>
       <zorder>ui_CurrentDiagram</zorder>
       <zorder>ui_treeView</zorder>
       <zorder>ui_Diagrams</zorder>
       <zorder>ui_CurrentDiagram</zorder>
       <zorder>ui_treeView</zorder>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Category</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QRadioButton" name="ui_categoryAll">
            <property name="text">
             <string>All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ui_categoryClass">
            <property name="text">
             <string>Class</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ui_categoryPackage">
            <property name="text">
             <string>Package</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ui_categoryInterface">
            <property name="text">
             <string>Interface</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ui_categoryOperation">
            <property name="text">
             <string>Operation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ui_categoryAttribute">
            <property name="text">
             <string>Attribute</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Search Term:</string>
         </property>
         <property name="buddy">
          <cstring>ui_searchTerm</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ui_searchTerm"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ui_searchTerm</tabstop>
  <tabstop>ui_treeView</tabstop>
  <tabstop>ui_CurrentDiagram</tabstop>
  <tabstop>ui_Diagrams</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="ui_buttonGroup"/>
 </buttongroups>
</ui>
